activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+*   Add option to index errors in nested attributes
+
+    For models which have nested attributes, errors within those models will
+    now be indexed if :index_errors is specified when defining a
+    has_many relationship, or if its set in the global config.
```

Comment:
No, that is the correct way to spell `its` here

On Mon, Oct 26, 2015, 5:20 PM Shunsuke Aida <notifications@github.com>
wrote:

> In activerecord/CHANGELOG.md
> <https://github.com/rails/rails/pull/19686#discussion_r43067124>:
>
> > @@ -1,3 +1,31 @@
> > +*   Add option to index errors in nested attributes
> > +
> > +    For models which have nested attributes, errors within those models will
> > +    now be indexed if :index_errors is specified when defining a
> > +    has_many relationship, or if its set in the global config.
>
> typo of it's?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/19686/files#r43067124>.
>